charter.html

```diff
@@ -281,13 +295,15 @@ <h2 id=work-items>Work Items</h2>
 <p>The Chairs may add Work Items, but must not add Work Items which lack
 the support of at least two <a href=#implementer>implementers</a>.
 
-<p class=note>Typically, Work Items begin life as a Proposal before
-being adopted.
+<p>Artifacts of Work Items may include an explainer describing the
+proposed changes to the web platform as well as a Draft Community Group
+Report defining the proposed changes in detail.
```

Comment:
Similarly:

```suggestion
proposed changes to the web platform as well as a Specification
defining the proposed changes in detail.
```

or "Community Group Report that is a Specification" as awkwardly used in https://www.w3.org/community/reports/.

Reply:
I think I've improved this in subsequent edits. Let me know if things are still unclear.